src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1173,7 +1173,8 @@ Glyphicon for success status messages
 .alert-success:before {
     font-family: 'Glyphicons Halflings';
     display: inline-block;
-    padding-right: 0.5em;
+    padding-top: .2em;
```

Comment:
@whipermr5 Is it really too much of a problem if the vertical alignment stays the way it is now? And if the `padding-top` approach works conveniently for both the status messages and the modals, would that be too much worse?